Applications/Games/Total War Rome II/Steam/script.js

```diff
@@ -13,6 +13,7 @@ new SteamScript()
     .appId(214950)
     .wineVersion(getLatestStagingVersion)
     .wineDistribution("staging")
+    .environment('{"__GL_THREADED_OPTIMIZATIONS": "1", "LD_PRELOAD" = "libpthread.so.0 libGL.so.1"}')
```

Comment:
Why is this required now?

Reply:
I saw a comment mentioning this.

Reply:
Where?

Reply:
https://github.com/PhoenicisOrg/scripts/blob/3a260960d1b424631c5b3c85bd3cb782204f144e/Applications/Games/Total%20War%20Rome%20II/Steam/script.js#L28

Reply:
Mh. I don't feel comfortable adding it in this PR.

Reply:
@ImperatorS79 do you think we can add this line to script?

Reply:
It would be better if someone could test it IMO.

Reply:
@ImperatorS79 can you do it?